js/tools.js

```diff
@@ -706,6 +706,18 @@ function getStorageAvailable() {
 	}
 }
 
+function refreshTotalCombinations(sign, number) {
+  let currentnumber;
+  //checking if parseInt throws exception
+  try {
+    currentnumber = parseInt($('.js-bulk-combinations-total').text()) + (sign * number);
+  }
+  catch (error) {
```

Comment:
If I'm not mistaken, `catch` should be in same line as the closing brace, like so:

```js
} catch (error) {
```

Reply:
In which case an exception would be thrown?

Reply:
I was trying to catch exception when `$('.js-bulk-combinations-total').text()` is null.
But you are right this won't throw exception in this case, so I will remove it.
Thanks !

Reply:
If the value is null or empty, `parseInt` will return `NaN`. If that's a possibility, you should consider handling (and testing) that case.